kaznet/settings/common.py

```diff
@@ -185,7 +185,7 @@
 ONA_PASSWORD = 'Password was here'
 ONA_MEMBERS_TEAM_ID = 102
 ONA_STATUS_FIELD = '_review_status'
-ONA_COMMENTS_FIELD = '_review_comments'
+ONA_COMMENTS_FIELD = '_review_comment'
```

Comment:
I am hesitant to make this change because of the possibility of a data migration.  Please undo this in this PR and create a different issue.

Reply:
@moshthepitt you should know that if this change is not made here then this issue will not have been fixed in this PR.

Reply:
Ok